src/index.ts

```diff
@@ -1,4 +1,13 @@
 import * as ko from 'knockout';
+import ModalVersion from './components/ModalVersion';
+
 require('knockout.validation');
-import 'bootstrap/dist/css/bootstrap.min.css';
-class AppViewModel {}
+
+ko.components.register('modal-version', ModalVersion);
+class AppViewModel {
+  version = function () {
+    window.$('#myModal').modal('show');
```

Comment:
If you create a separated component for the navBar, this code is going to be there.